pyuvdata/uvbase.py

```diff
@@ -62,12 +62,14 @@ def __init__(self):
 
         # String to add to history of any files written with this version of pyuvdata
         self.pyuvdata_version_str = ('  Read/written with pyuvdata version: '
-                                     + uvversion.version + '.')
-        if uvversion.git_hash != '':
-            self.pyuvdata_version_str += ('  Git origin: ' + uvversion.git_origin
-                                          + '.  Git hash: ' + uvversion.git_hash
-                                          + '.  Git branch: ' + uvversion.git_branch
-                                          + '.  Git description: ' + uvversion.git_description + '.')
+                                     + __version__ + '.')
+
+        # remove the following because __version__ has this info in it if available.
+        # if uvversion.git_hash != '':
+        #     self.pyuvdata_version_str += ('  Git origin: ' + uvversion.git_origin
+        #                                   + '.  Git hash: ' + uvversion.git_hash
+        #                                   + '.  Git branch: ' + uvversion.git_branch
+        #                                   + '.  Git description: ' + uvversion.git_description + '.')
```

Comment:
would also say the commented lines here should just be removed.

Reply:
Done